src/main/java/io/sniffy/servlet/RequestStats.java

```diff
@@ -1,26 +1,35 @@
 package io.sniffy.servlet;
 
+import io.sniffy.socket.SocketMetaData;
+import io.sniffy.socket.SocketStats;
 import io.sniffy.sql.StatementMetaData;
 
 import java.util.List;
+import java.util.Map;
 
 class RequestStats {
 
     private long timeToFirstByte;
     private long elapsedTime;
     private List<StatementMetaData> executedStatements;
+    Map<SocketMetaData, SocketStats> socketOperations;
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: [Use explicit scoping instead of the default package private level](https://www.codacy.com/app/sniffy/sniffy/file/2498344323/issues/source?bid=3005960&fileBranchId=3034483#l15)